_internals/configuration.md

```diff
@@ -118,329 +118,329 @@ When `ANSI_DEFAULTS` is enabled, it enables the following ISO settings:
 
 
-### babelfishpg_tsql.datefirst
+### `babelfishpg_tsql.datefirst`
 
 Sets the first day of the week to a number from 1 through 7.
 
 
-### babelfishpg_tsql.default_locale
+### `babelfishpg_tsql.default_locale`
 
 The default locale to use when creating a new collation. The default value is
-`en_US`. Only superusers can set this variable. 
+`en_US`. Only superusers can set this variable.
 
 
-### babelfishpg_tsql.disable_batch_auto_commit
+### `babelfishpg_tsql.disable_batch_auto_commit`
 
 Disable auto commit inside procedures. The default value is `false`.
 
 
-### babelfishpg_tsql.disable_internal_savepoint
+### `babelfishpg_tsql.disable_internal_savepoint`
 
 Disable internal savepoints. The default value is `false`.
 What is the purpose of this variable? It controls whether a savepoint is created to
 handle undo if a command fails. Savepoints are not started for batch commands, as
 error handling must be taken care of at the statement level.
 
 
-### babelfishpg_tsql.disable_txn_in_triggers
+### `babelfishpg_tsql.disable_txn_in_triggers`
 
-Disable transactions in triggers. The default value is `false`. 
+Disable transactions in triggers. The default value is `false`.
 TSQL triggers terminate if there is no transaction active at the end.
 
 
-### babelfishpg_tsql.dump_antlr_query_graph
+### `babelfishpg_tsql.dump_antlr_query_graph`
 
 Dump query graph parsed by ANTLR parser to local disk. The path used is
 `/tmp/antlr.dot`.
 
 
-### babelfishpg_tsql.enable_antlr_detailed_log
+### `babelfishpg_tsql.enable_antlr_detailed_log`
 
 Enable detailed ATNLR parser logging.
 
 
-### babelfishpg_tsql.enable_ownership_structure
+### `babelfishpg_tsql.enable_ownership_structure`
 
-Enable Babelfish ownership structure. 
+Enable Babelfish ownership structure.
 
 
-### babelfishpg_tsql.escape_hatch_compatibility_level
+### `babelfishpg_tsql.escape_hatch_compatibility_level`
 
 Defines compatibility level.
 
 
-### babelfishpg_tsql.escape_hatch_constraint_name_for_default
+### `babelfishpg_tsql.escape_hatch_constraint_name_for_default`
 
-Escape hatch for `DEFAULT` option in <code>ALTER TABLE ADD</code> constraint
+Escape hatch for `DEFAULT` option in `ALTER TABLE ADD` constraint
 
 
-### babelfishpg_tsql.escape_hatch_database_misc_options
+### `babelfishpg_tsql.escape_hatch_database_misc_options`
 
-Escape hatch for misc options in <code>CREATE/ALTER DATABASE</code>
+Escape hatch for misc options in `CREATE/ALTER DATABASE`
 
 
-### babelfishpg_tsql.escape_hatch_for_replication
+### `babelfishpg_tsql.escape_hatch_for_replication`
 
-Escape hatch for <code>[NOT] FOR REPLICATION</code> option
+Escape hatch for `[NOT] FOR REPLICATION` option
 
 
-### babelfishpg_tsql.escape_hatch_fulltext
+### `babelfishpg_tsql.escape_hatch_fulltext`
 
 Escape hatch for fulltext search.
 
 
-### babelfishpg_tsql.escape_hatch_index_clustering
+### `babelfishpg_tsql.escape_hatch_index_clustering`
 
 Escape hatch for `CLUSTERED` option in `CREATE INDEX`
```

Comment:
Not just in CREATE INDEX, also applies to constraints. Do not add any explanation for escape hatches, but point to the escape hatch section, where the explanation should be included.

Reply:
I have added the constraints.

I have added a link to the "limitations" documentation for more information about hatches.